src/main/java/uk/gov/justice/laa/crime/cfecrime/cma/enums/AssessmentType.java

```diff
@@ -1,4 +1,38 @@
 package uk.gov.justice.laa.crime.cfecrime.cma.enums;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
+import com.fasterxml.jackson.annotation.JsonValue;
+import javax.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.stream.Stream;
+/**
+ * copied from crime-means-assessment project
+ */
+@Getter
+@AllArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
 public enum AssessmentType {
+    INIT("INIT"), FULL("FULL");
+
+    @NotNull
+    @JsonPropertyDescription("Determines the assessment type")
+    private String type;
+
+    @JsonValue
+    public String getType() {
+        return type;
+    }
```

Comment:
Think this is the wrong assessment type enum - the stateless one has 'INIT' and 'BOTH' as options. Maybe we should keep the package names the same as we copy the files over to avoid confusion?